README.md

```diff
@@ -1,79 +1,41 @@
-# ![Islandora 8](https://camo.githubusercontent.com/738dd7cbd90a3ef06b9bb55a4cf5ed385a048fd4/687474703a2f2f69736c616e646f72612e63612f73697465732f64656661756c742f66696c65732f696d616765732f6c6f6273746572434c41572e706e67) Islandora 8 Documentation
+# Islandora 8 Documentation
+![Islandora 8](https://camo.githubusercontent.com/738dd7cbd90a3ef06b9bb55a4cf5ed385a048fd4/687474703a2f2f69736c616e646f72612e63612f73697465732f64656661756c742f66696c65732f696d616765732f6c6f6273746572434c41572e706e67)
 
-## About this repository (github.com/Islandora/documentation)
+[Read the documentation online](https://islandora.github.io/documentation/)
 
-This repository contains the user, developer, and administrator documentation for the [Islandora project](https://islandora.ca/), for versions 8 and above. This documentation is available to read at https://islandora.github.io/documentation/, where it is automatically deployed on commit from the source code at https://github.com/Islandora/documentation. Documentation for Islandora 7.x and previous versions is available at https://wiki.lyrasis.org/display/ISLANDORA/. 
```

Comment:
I would suggest to start the README with the information that this repository serves a 3 very distinct purposes (as is outlined in the repo's "About" text):

1. it contains the documentation source code for Islandora 8
2. its Wiki contains the minutes for Tech calls and User calls
3. it hosts the central issue queue for the entire Islandora 8 project

Reply:
+1 I think those three points summarize the repo nicely